tools/debian_kernel_postinst_lunatik.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Can you please add license here? Dual MIT/GPL2 is preferable.. thanks!

Reply:
Just done, with copyright assignment so that you may change it at your will.